### PR TITLE
fix issues on azure- terraform

### PR DESCRIPTION
**Pull request by shahinshirvani**

### CODE REVIEW

File: azure/terraform/registrieswebhooks.rego [hunk 1]
```diff
@@ -34,16 +34,15 @@ acr_webhooks = false {
     azure_issue["acr_webhooks"]
 }
 
-acr_webhooks_err = "azurerm_container_registry_webhook property 'service_uri' need to be exist. Its missing from the resource." {
-    azure_attribute_absence["acr_webhooks"]
-} else = "Azure ACR currently does not have HTTPS protocol enabled for webhook" {
-    azure_issue["acr_webhooks"]
+acr_webhooks_miss_err = "azurerm_container_registry_webhook property 'service_uri' need to be exist. Its missing from the resource." {
```
Comment (rezoan) [inline azure/terraform/registrieswebhooks.rego:37]: @shahinshirvani any reason not to include this in a else block with `acr_webhooks_err `?

File: azure/terraform/storageaccounts.rego [hunk 2]
```diff
@@ -23,12 +23,12 @@ azure_issue["storage_secure"] {
 
 storage_secure {
     lower(input.resources[_].type) == "azurerm_storage_account"
-    not azure_attribute_absence["storage_secure"]
```
Comment (rezoan) [inline azure/terraform/storageaccounts.rego:26]: @shahinshirvani this line does not hurt if exist. removing this line also work as default is true.

**Review by rezoan — CHANGES_REQUESTED** [review]

File: azure/terraform/storageaccounts.rego [hunk 3]
```diff
@@ -23,12 +23,12 @@ azure_issue["storage_secure"] {
...
     not azure_issue["storage_secure"]
 }
 
 storage_secure {
     azure_attribute_absence["storage_secure"]
+    not azure_issue["storage_secure"]
```
Comment (rezoan) [inline azure/terraform/storageaccounts.rego:31]: this is the fix.

File: azure/terraform/registrieswebhooks.rego (hunk 1 above)
Comment (shahinshirvani) [reply azure/terraform/registrieswebhooks.rego]: @rezoan fixed
at first I didn't know about this kind of new error message

File: azure/terraform/storageaccounts.rego (hunk 2 above)
Comment (shahinshirvani) [reply azure/terraform/storageaccounts.rego:26]: @rezoan 
in this case we need that
assume that ```enable_https_traffic_only``` exists and the value of that is ```false```


```not resource.properties.enable_https_traffic_only``` checks ```enable_https_traffic_only``` didn't exist or the of that is ```false```
so ```azure_attribute_absence["storage_secure"]``` will be true
and ```not azure_issue["storage_secure"]``` will be false

so
```
storage_secure {
    azure_attribute_absence["storage_secure"]
    not azure_issue["storage_secure"]
}
```
has no answer

**Review by rezoan — APPROVED** [review]

**Pull request merged**